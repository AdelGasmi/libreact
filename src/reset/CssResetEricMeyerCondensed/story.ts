import {createElement as h} from 'react';
import {storiesOf} from '@storybook/react';
import {action} from '@storybook/addon-actions';
import {linkTo} from '@storybook/addon-links';
import CssResetEricMeyerCondensed from '.';

storiesOf('reset', module)
  .add('CssResetEricMeyerCondensed', () =>
    h(CssResetEricMeyerCondensed)
  );
