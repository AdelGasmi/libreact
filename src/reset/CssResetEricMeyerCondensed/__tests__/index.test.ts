import {createElement as h} from 'react';
import {renderToStaticMarkup} from 'react-dom/server';
import CssResetEricMeyerCondensed from '..';

describe('<CssResetEricMeyerCondensed>', () => {
  it('renders as expected', () => {
    const html = renderToStaticMarkup(h(CssResetEricMeyerCondensed));

    expect(html).toMatchSnapshot();
  });
});
