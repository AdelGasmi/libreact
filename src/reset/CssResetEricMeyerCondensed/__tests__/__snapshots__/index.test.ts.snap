// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CssResetEricMeyerCondensed> renders as expected 1`] = `
"<style>body,div,dl,dt,dd,ul,ol,li,h1,h2,h3,h4,h5,h6,pre,form,fieldset,input,textarea,p,blockquote,th,td {
    padding: 0;
    margin: 0;
}
fieldset, img {
    border: 0;
}
table {
    border-collapse: collapse;
    border-spacing: 0;
}
ol, ul {
    list-style: none;
}
address, caption, cite, code, dfn, em, strong, th, var {
    font-weight: normal;
    font-style: normal;
}
caption, th {
    text-align: left;
}
h1, h2, h3, h4, h5, h6 {
    font-weight: normal;
    font-style: 100%;
}
q:before, q:after {
    content: '';
}
abbr, acronym {
    border: 0;
}
</style>"
`;
